app/views/govuk_publishing_components/components/_radio.html.erb

```diff
@@ -29,17 +39,29 @@
         <% if label_hint_id %>
           aria-describedby="<%= label_hint_id %>"
         <% end %>
+        <% if conditional_id %>
+          data-aria-controls="<%= conditional_id %>"
+        <% end %>
       >
       <%= render "govuk_publishing_components/components/label", {
         hint_id: label_hint_id,
         html_for: label_id,
-        classes: 'gem-c-radio__label',
+        classes: 'govuk-label govuk-radios__label',
         text_classes: 'gem-c-radio__label__text',
         hint_text_classes: 'gem-c-radio__label__hint',
         hint_text: item[:hint_text],
         text: item[:text],
         bold: item[:bold]
       } %>
     </div>
+    <% if item[:conditional] %>
+    <div class="govuk-radios__conditional" id="<%= conditional_id %>">
+      <%= item[:conditional] %>
+    </div>
+    <% end %>
   <% end %>
 <% end %>
+<% if items.any? %>
```

Comment:
I'm not sure that having opening and closing divs in separate IF statements is a good approach. Have you considered using content_tag and a helper method to wrap some of this code?

Reply:
I can use `content_tag` if that makes it easier to read

Reply:
@andysellick what's the advantage of using the `content_tag` with a helper?

Reply:
I assumed that as soon as you started using content_tag and passing in chunks of code the view would start to get overly complicated, at which point it would be good to move some of that code into a helper. The [step by step component](https://github.com/alphagov/govuk_publishing_components/blob/master/lib/govuk_publishing_components/presenters/step_by_step_nav_helper.rb) does some nesting of elements like this, if that helps.

Reply:
This is solved in GOV.UK Frontend by making the fieldset component an additional component then using the nesting interface 'caller'.

In ERB this can be achieved with 'yield'

something like

```
fieldset do
  radios
end
```

Reply:
@nickcolley this is what I want to achieve, but I'm trying to do the minimum in this PR to update the radios and make them use Frontend then I'm planning to address the labels, hints and fieltset in separate PRs